leankyr/Easy_Problems/Caesar_Cipher/caesarCipher.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3.5
+# -*- coding: utf-8 -*-
+
+import sys
+
+def caesarCipher(s, k):
+    k = k%26
+    # Caps2integers Mappin
+    Caps2integers = dict(zip("ABCDEFGHIJKLMNOPQRSTUVWXYZ",range(0,26)))
+    integers2Caps = dict(zip(range(0,26),"ABCDEFGHIJKLMNOPQRSTUVWXYZ"))
+    
+    letters2integers = dict(zip("abcdefghijklmnopqrstuvwxyz",range(0,26)))
+    integers2letters = dict(zip(range(0,26),"abcdefghijklmnopqrstuvwxyz"))
```

Comment:
Instead of mapping all letters to numbers in order to make use of the mod, try solving it using the deque data structure, this way you can understand how it works and why it might be useful in some cases